fre/app/freapp.py

```diff
@@ -7,70 +7,118 @@
 
 from .mask_atmos_plevel import mask_atmos_plevel_subtool
 from .generate_time_averages.generate_time_averages import generate
+from .regrid_xy.regrid_xy import _regrid_xy
 
 @click.group(help=click.style(" - access fre app subcommands", fg=(250,154,90)))
 def app_cli():
     ''' entry point to fre app click commands '''
 
+@app_cli.command()
+@click.option("-i", "--input_dir",
+              type = str,
+              help = "`inputDir` / `input_dir` (env var) specifies input directory to regrid, " + \
+                     "typically an untarredv history file archive" ,
+              required = True)
+@click.option("-o", "--output_dir",
+              type = str,
+              help = "`outputDir` / `output_dir` (env var) specifies target location for output" + \
+                     " regridded files",
+              required = True)
+@click.option("-b", "--begin",
+              type = str,
+              help = "`begin` / `begin` (env var) ISO8601 datetime format specification for" + \
+                     " starting date of data, part of input target file name",
+              required = True)
+@click.option("-t", "--tmp_dir",
```

Comment:
It's pretty easy to use options like this (one letter) but I also feel like some of these can have more explicit or different option flags (the first `-[option]` at least). Maybe this could be `-tmp` because `-t` is used in other places too, referencing things like `time` or `target`. It could get confusing with `-t` options that refer to a bunch of things. Thoughts?

Reply:
change implemented in next commit

Reply:
i agree with Dana. Bronx has long had an annoying ambiguity between -t being --target and --time, and it's worth keeping the the options consistent whereever possible.

How about we outlaw the short options altogether except in especially well-known cases?